inc/class-command.php

```diff
@@ -268,6 +294,58 @@ protected function provision( InputInterface $input, OutputInterface $output ) {
 		return $this->run_command( 'vagrant provision' );
 	}
 
+	/**
+	 * Command to upgrade chassis and re provision the VM.
+	 *
+	 * @param InputInterface $input
+	 * @param OutputInterface $output
+	 * @return int Status code to return.
+	 */
+	protected function upgrade( InputInterface $input, OutputInterface $output ) {
+		$output->writeln( '<info>Updating chassis...</>' );
```

Comment:
```suggestion
		$output->writeln( '<info>Updating Chassis...</>' );
```

Reply:
I need to update the docs anyway so yeah I can do that.